mirrormanager2/lib/repomap.py

```diff
@@ -97,47 +96,62 @@ def repo_prefix(path, category, ver):
                     prefix = u'fedora-install-%s' % version
-            elif isModular and isServer:
-                if isBikeshed:
-                    # Modular Bikeshed
+            elif isModular:
+                if isUpdatesReleased:
+                    # updates-released-modular-
                     if isDebug:
-                        prefix = u'modular-bikeshed-server-debug'
+                        prefix = u'updates-released-modular-debug-f%s' % version
                     elif isSource:
-                        prefix = u'modular-bikeshed-server-source'
+                        prefix = u'updates-released-modular-source-f%s' % version
                     else:
-                        prefix = u'modular-bikeshed-server'
-                elif isUpdatesReleased:
-                    # Modular Server updates-released-
-                    if isDebug:
-                        prefix = u'modular-server-updates-released-debug-f%s' % version
-                    elif isSource:
-                        prefix = u'modular-server-updates-released-source-f%s' % version
-                    else:
-                        prefix = u'modular-server-updates-released-f%s' % version
+                        prefix = u'updates-released-modular-f%s' % version
                 elif isUpdatesTesting:
-                    # Modular Server updates-testing
+                    # updates-testing-modular-
                     if isDebug:
-                        prefix = u'modular-server-updates-testing-debug-f%s' % version
+                        prefix = u'updates-testing-modular-debug-f%s' % version
                     elif isSource:
-                        prefix = u'modular-server-updates-testing-source-f%s' % version
+                        prefix = u'updates-testing-modular-source-f%s' % version
                     else:
-                        prefix = u'modular-server-updates-testing-f%s' % version
-                else:
-                    #Modular Releases
+                        prefix = u'updates-testing-modular-f%s' % version
+                elif isRawhide:
+                    # rawhide-modular
                     if isDebug:
-                        prefix = u'modular-fedora-server-debug-f%s' % version
+                        prefix = u'rawhide-modular-debug'
                     elif isSource:
-                        prefix = u'modular-fedora-server-source-f%s' % version
+                        prefix = u'rawhide-modular-source'
                     else:
-                        prefix = u'modular-fedora-server-f%s' % version
+                        prefix = u'rawhide-modular'
+                # fedora-modular-
+                elif isDebug:
+                    prefix = u'fedora-modular-debug-%s' % version
+                elif isSource:
+                    prefix = u'fedora-modular-source-%s' % version
+                else:
+                    prefix=u'fedora-modular-%s' % version
+            elif isUpdatesReleased:
```

Comment:
This entire portion is for the use case when we have a Everything tree inside of updates and updates-testing  which we will have next to Modular for updates and updates-testing